CHANGELOG.unreleased.md

```diff
@@ -27,4 +29,5 @@
 
 <!--packages-start-->
 
+- xo-server patch
```

Comment:
There should be a blank line after this line.

It appears Prettier did not format automatically this file, is there an issue with the current configuration or is it normal?

Reply:
I am not sure, the pre commit hooks ran without problem